src/exceptions/UserServiceError.js

```diff
@@ -0,0 +1,5 @@
+module.exports = class UserServiceException extends Error {
```

Comment:
Yeah, it's a generic exception, but with a custom name - kinda might come in handy at some point